src/components/footer/config/index.ts

```diff
@@ -0,0 +1,656 @@
+import { LinkConfig } from './link';
+
+declare global {
+  interface Window {
+    OneTrust: {
+      ToggleInfoDisplay: () => void;
+    };
```

Comment:
In the project itself, how are we going to link OneTrust? Is that line enough?

Reply:
So that line is the type definition for the global object, the function that calls one trust is there https://github.com/smg-automotive/components-pkg/pull/349/files#diff-936dd87b2b01691a3c76c0a1b332e62cca56304a1107179a99f242b25b402d3eR386

Reply:
Yes 👍 
Here you just set a new property on `Window`?

Reply:
Yap. So it's not setting anything it's adding a global type definition. By default in the browser, the global scope is the window object which is typed as something like `unknown`. In this case, the OneTrust-CMP script will add functions to that global scope that can be called from anywhere. This part makes typescript understand that we expect that behaviour. (It does not have any effect on the code)

https://www.typescriptlang.org/docs/handbook/declaration-files/templates/global-d-ts.html